apps/web-giddh/src/app/shared/header/components/account-add-new-details/account-add-new-details.component.ts

```diff
@@ -432,7 +448,11 @@ export class AccountAddNewDetailsComponent implements OnInit, OnChanges, AfterVi
         }
     }
 
-    public getStateCode(gstForm: FormGroup, statesEle: ShSelectComponent) {
+    public getStateCode(gstForm: FormGroup, statesEle: ShSelectComponent, event: KeyboardEvent) {
+        const keyAvoid = [9, 37, 38, 39, 40];
+        if (keyAvoid.findIndex(x => x === event.keyCode) > -1) {
```

Comment:
`keyCode` is deprecated use `key` instead and please avoid use of single character variable name 'x'.
Do make sure that the user can move between items when arrow keys are pressed as the current functionality works